proving/proving_test.go

```diff
@@ -166,14 +164,14 @@ func Test_Generate_TestNetSettings(t *testing.T) {
 	ch := make(shared.Challenge, 32)
 	cfg := config.DefaultConfig()
 
-	// https://colab.research.google.com/github/spacemeshos/notebooks/blob/main/post-proof-params.ipynb
-	cfg.LabelsPerUnit = 2 << 16
-	cfg.B = 16
-	cfg.K1 = 279
-	cfg.K2 = 287
-	cfg.N = 24
+	// Test-net settings:
+	cfg.LabelsPerUnit = 20 * 1024 / 16 // 20kB unit
```

Comment:
With the suggestion above this could be:

```suggestion
	cfg.LabelsPerUnit = 20 * KiB / (16*B) // 20kB unit
```